src/custom/pages/Profile/index.tsx

```diff
@@ -30,7 +30,7 @@ export default function Profile() {
   const { account, chainId } = useActiveWeb3React()
   const { profileData, isLoading, error } = useFetchProfile()
   const lastUpdated = useTimeAgo(profileData?.lastUpdated)
-
+  const isTTradesTooltipVisible = account && chainId == 1 && parseInt(formatInt(profileData?.totalTrades)) > 0
```

Comment:
There's a typo in the variable name.
Also, you can simplify the last part with `!!profileData?.totalTrades`

Reply:
done